### PR TITLE
Bedpetobed12

### PR DESCRIPTION
**Pull request by ernfrid**
This cleans up the bedpetobed12 a bit. The only meaningful change is for the chunkStarts for positive strand, non-BND variants. They have changed by 1. This is consistent with the original behavior in the legacy branch.

### CODE REVIEW

File: svtools/bedpetobed12.py [hunk 1]
```diff
@@ -3,89 +3,186 @@
...
+                bedpe.e2,
+                name, 
+                bedpe.score,
+                bedpe.o2,
+                color))
+        return output_lines
+
 def processBEDPE(bedpe_stream, name, dist, output_handle=sys.stdout):
     #Process the BEDPE file and convert each entry to SAM.
-    if name is not None or bedpe_stream == sys.stdin:
```
Comment (apregier) [inline svtools/bedpetobed12.py]: You are losing this check - I don't know if it was meaningful or not?

Comment (ernfrid) [inline svtools/bedpetobed12.py]: I came to the conclusion that since we had a default name set at the command line that this was not longer relevant.

**Comment by apregier** [comment]
I wasn't able to find anything, although I don't have a lot of confidence in my ability to detect python bugs.  It does seem like you have improved the code.  The refactoring would have been easier to review if you had put it in a lot of small commits with really specific messages - like "extract method to get track color".  Just something to keep in mind for the future.

**Comment by ernfrid** [comment]
Thanks Allison!

**Pull request merged**